docs/examples/polygonal_poinset_reconstruction.py

```diff
@@ -0,0 +1,60 @@
+from pathlib import Path
+from compas.geometry import Pointcloud, Point, Line, Polyhedron, Polygon
+from compas_view2.app import App
+# from compas_view2.collections import Collection
+from compas.colors import Color
+from compas_cgal.reconstruction import pointset_normal_estimation
+from compas_cgal.polygonal_surface_reconstruction import polygonal_surface_reconstruction_ransac
+
+# Define the file path for the point cloud data
+FILE = Path(__file__).parent.parent.parent / "data" / "two_intersecting_boxes.ply"
+
+# Load the point cloud data from the PLY file
+cloud = Pointcloud.from_ply(FILE)
+
+# Estimate normals for the point cloud
+points, vectors = pointset_normal_estimation(cloud, 4, True)
+vertices, faces = polygonal_surface_reconstruction_ransac(points, vectors)
+
+# Create polygons from vertices and faces
+polygons = [Polygon([Point(*vertices[i]) for i in face]) for face in faces]
+
+# Create a polyhedron from vertices and faces
+shape = Polyhedron(vertices.tolist(), faces.tolist())  # revise the Shape API
+
+# Create a viewer
+viewer = App(width=1600, height=900)
+viewer.view.camera.position = [5, -4, 2]
+viewer.view.camera.look_at([0, 1, 0])
```

Comment:
i would move this to the end, into a "visualization" block. this makes it clearer where the algorithm part is and where the viz...